tests/quick_tests/run.cmake

```diff
@@ -22,7 +22,7 @@ SEPARATE_ARGUMENTS(ALL_TESTS)
 EXECUTE_PROCESS(COMMAND ${CMAKE_CTEST_COMMAND} --force-new-ctest-process --output-on-failure -O quicktests.log RESULT_VARIABLE res_var)
 
 if(NOT "${res_var}" STREQUAL "0")
-  MESSAGE( "
+  MESSAGE(SEND_ERROR "
 
 *******************************     WARNING     *******************************
```

Comment:
I don't know if that's what we want to say. I was just raising the question that it looks odd to issue an error, but then print `*** WARNING ***` to the screen. It seemed like a question worth discussing :-)